ppl/src/test/java/com/amazon/opendistroforelasticsearch/sql/ppl/parser/AstBuilderTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.ppl.parser;
+
+import com.amazon.opendistroforelasticsearch.sql.ppl.antlr.PPLSyntaxParser;
+import com.amazon.opendistroforelasticsearch.sql.ppl.node.Node;
+import java.util.Arrays;
+import java.util.Collections;
+import org.junit.Test;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.agg;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.aggregate;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.equalTo;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.filter;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.function;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.intLiteral;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.map;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.project;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.relation;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.stringLiteral;
+import static com.amazon.opendistroforelasticsearch.sql.ppl.plans.dsl.DSL.unresolvedAttr;
+import static org.junit.Assert.assertEquals;
+
+public class AstBuilderTest {
```

Comment:
could u add test case for qualified index name, "log.2020.04.20."

Reply:
Ok sure, one question: shall we unquote the (single/double/backticks) quoted identifiers in this module?